doc/onboarding/model/user.md

```diff
@@ -0,0 +1,22 @@
+# `user` state
+
+
+## Definition
+```
+{
+	<string> name,
+	<id> id,
+	<email> email,
+	<boolean> is_registered: false
+	<boolean> is_shared: false
```

Comment:
After reflexions, using flag is not the best solution to know if the onboarding should be launched or not. I think it could be better to store an array of steps to be submitted by the user. By default this array contains all steps like:

```
[<string>] onboarding_steps: ['welcome',  'agreement', 'password', 'confirmation']
```

If we want the user to resubmit agreements for exemple, we have just to had this step in this array.
Of course, the order of all steps should be stored in the proxy.

What do you think about?